CMakeLists.txt

```diff
@@ -16,7 +16,7 @@ endif()
 
 option(SHADERC_SKIP_SPVC "Skip libshaderc_spvc" ${SHADERC_SKIP_SPVC})
 if(NOT ${SHADERC_SKIP_SPVC})
-  set(SHADERC_ENABLE_SPVC ON)
+  set(SHADERC_ENABLE_SPVC OFF)
```

Comment:
This doesn't work for me in a tree without the spirv-cross sources.
If I give none of the options SHADERC_ENABLE_SPVC or SHADERC_SKIP_SPVC then it fails with error:

CMake Error at third_party/shaderc/third_party/CMakeLists.txt:91 (message):
  SPIRV-Cross was not found - required for compilation

Just negating the setting isn't right.